src/bin/uhyve.rs

```diff
@@ -283,30 +283,32 @@ fn main() {
 						error!("CPU {} crashes! {}", tid, x);
 						None
```

Comment:
Is `std::process::exit` (and potentially the rftrace dump) not required in error case, because all threads are guaranteed to exit in that case?

Reply:
You are right. I fixed it.